### ISSUE TITLE
Svi Cloud 8P Installation Guide Yogurt TV, Yogurt Kids, Yogurt Malaysia & 合家欢

### ISSUE THREAD

**Issue by plsharevme** [plsharevme/note#16]
## Step to install SVI official APK

After get the box " SVI 8P" just connect HDMI to TV ➡️ 📺  , Connect to WIFI or Lan cable 📶

Follow the Picture to installation SVI Live & Movies, Adult Apk
- Click Orz and typing the following url to install.
- If you cannot find Orz, just press 6868 at remote control and install it.

#### Yogurt TV, Yogurt Kids, Yogurt Malaysia & 合家欢
> keyin 6868c.cc and install above mention apk.

#### Adult APK (18+) Cherry TV🧑 
> keyin 8989c.cc and install Cherry TV, password if need 898989.

![image1](https://user-images.githubusercontent.com/64370408/153214281-b465f84c-6124-407b-87d6-54707bbe5c2a.jpg)
![image2](https://user-images.githubusercontent.com/64370408/153214307-8e1a497d-9d21-4a62-94e6-52a997aa8742.jpg)
![image4](https://user-images.githubusercontent.com/64370408/153212731-8aa0707f-f4f5-448f-9570-b113e31c6677.jpg)
![image6](https://user-images.githubusercontent.com/64370408/153213718-e4006898-bf91-41e9-b8d1-94f185820ca3.jpg)

**Comment by singsing89** [comment]
Nice

**Comment by ardmorepark** [comment]
What is the authorization code?

**Comment by bumana1974** [comment]
Yes authorized code

**Comment by ardmorepark** [comment]
What is authorization code?Best regardsIndra WidjajaOn 16 Mar 2024, at 09.25, bumana1974 ***@***.***> wrote:﻿
Yes authorized code

**Comment by fuham900** [comment]
What is the authorization code?

**Comment by ericvlog** [comment]
> What is the authorization code?

Is adult channel installation code is 6868 or 1111 , other code need ask your seller.

**Comment by xfinalangelx** [comment]
Anyone encountered L001-000-BRE Request failed (E110005)?

**Comment by zhuhang-jasper** [comment]
> Anyone encountered L001-000-BRE Request failed (E110005)?

Yes, seems the 8p model no longer works? Scammer company, 2023 model they killed and the WhatsApp support QR code is invalid.
